R/AddModules.R

```diff
@@ -1,30 +1,51 @@
 #' @name AddModules
+#'
 #' @title AddModules
 #' @description { This function adds a module to a shiny app created by BaSS.  }
 #'
 #' @param vModuleIDs IDs of the shiny module to be added to the application. The source code for each module should be saved in strModuleDirectory with files named "mod_{strModuleID}UI.R" and mod_{strModuleID}Server.R". If desired, submodules may also be included using the following naming convention "mod_{strModuleID}_XXX.R".
+#' @param strModuleID alias for vModuleIDs. Items are merged if both are provided.
```

Comment:
I am confused as to what the vModuleIDs do and strModuleID are going to do.   Not sure how I add a sub-module? Are sub-modules added if I give more than one in the vector?

Reply:
Yeah, it's redundant. strModuleID is just added to the vModuleIDs list. Don't really need both though. Will clean it up.